docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -68,21 +99,18 @@ The Alert details flyout contains the following informational tabs:
 [discrete]
 *Overview*
 
-The *Overview* tab provides an overview of the alert and shows relevant threat intelligence details. Use this information to analyze what generated the alert, so you can triage its relevance and resolve it as appropriate.
-
-The *Overview* tab has three main sections: *Reason*, *Document Summary*, and *Enriched data*. The *Reason* and *Document Summary* sections are standard for every alert. The *Enriched data* section is conditional and only displays for alerts with intelligence data.
+The *Overview* tab provides an overview of the alert and shows relevant threat intelligence details. Use this information to understand what generated the alert so you can appropriately triage and resolve it.
 
-The *Reason* section describes what generated the alert and provides general alert details. Use this information to understand the alert's origin and determine if the alert is relevant to your investigation.
-
-TIP: To view more details about the rule that produced the alert, click *View Rule detail page*.
+The *Overview* tab has three main sections: the summary alert summary, *Highlighted Fields*, and *Enriched data*. The alert summary and *Highlighted Fields* sections are standard for every alert. The *Enriched data* section only displays for alerts with intelligence data.
```

Comment:
```suggestion
The *Overview* tab has three main sections: the alert summary, *Highlighted Fields*, and *Enriched data*. The alert summary and *Highlighted Fields* sections are standard for every alert. The *Enriched data* section only displays for alerts with intelligence data.
```

Reply:
My only thing here is while the *Highlighted Fields* section is present for every alert, the fields aren't standard between alerts as the fields shown are based on the rule type and available data. Unless I missed something and something changed. Maybe:

" The alert summary section is standard for every alert. The Highlighted Fields section will show data certain fields based on rule type as well as the underlying data. ...."

@janmonschke would be able to help us clarify for sure

Reply:
Ignore me, missed the comment below: `The *Highlighted Fields* section is also provided for all alerts and is customized to only show the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.` 

Thanks!